app/assets/javascripts/discourse/app/components/modal/bookmark.hbs

```diff
@@ -8,6 +8,15 @@
   data-bookmark-id={{this.bookmark.id}}
   {{did-insert this.didInsert}}
 >
+  <:headerPrimaryAction>
```

Comment:
This is the new block which has to be manually define in each modals where we want to support this

Reply:
What happens if they don't define it? They still get the swipe up/down UI? Or they stick with the old design?